libs/blocks/vimeo/vimeo.js

```diff
@@ -79,7 +82,7 @@ class LiteVimeo extends HTMLElement {
 }
 
 export default async function init(a) {
-  if (isInTextNode(a)) return;
+  if (isInTextNode(a) || !a) return;
```

Comment:
Would it be better to first check for `a` before applying a function to it?
```suggestion
  if (!a || isInTextNode(a)) return;
```